apps/osvr_server.cpp

```diff
@@ -50,16 +57,84 @@ void handleShutdown() {
 int main(int argc, char *argv[]) {
     auto log = ::osvr::util::log::make_logger(OSVR_SERVER_LOG);
 
-    std::string configName(osvr::server::getDefaultConfigFilename());
-    if (argc > 1) {
-        configName = argv[1];
+    std::string configName; // server configuration filename
+
+    opt::options_description optionsAll("All Options");
+    opt::options_description optionsVisible("Command Line Options");
+    opt::positional_options_description optionsPositional;
+
+    optionsPositional.add("config", -1);
```

Comment:
Yes. So if you run `osvr_server blah.json` then it will use `blah.json` as the configuration file. The same thing happens if you drag and drop `blah.json` onto the `osvr_server.exe` program in Windows.